packages/e2e/detox/init.ts

```diff
@@ -16,7 +15,7 @@ dotenv.config();
 
 // Cucumber has default timeout of 5000, not enough for Detox async operations
 // https://wix.github.io/Detox/docs/guide/cucumber-js-integration
-setDefaultTimeout(120 * 1000);
+setDefaultTimeout(600000);
```

Comment:
Assuming milliseconds here, 10 minutes seems like an extraordinary amount of time to set as the default timeout?

Reply:
Based on several runs on my fork anything under 5000000 (8min) produces flakiness. Initial app bundling time is what seems to be the major contributor. That takes a few minutes on my machine as well.

Reply:
Cool, could you add a comment explaining this for historical context?